git-branchless/src/revset/eval.rs

```diff
@@ -107,6 +108,20 @@ pub enum EvalError {
     #[error("expected {expr} to evaluate to 1 element, but got {actual_len}")]
     ExpectedSingletonSet { expr: String, actual_len: usize },
 
+    #[error(
+        "expected '{expr}' to evaluate to {}, but got {actual_len}",
+        Pluralize {
+            determiner: None,
+            amount: *expected_len,
+            unit: ("element", "elements"),
+        }
+    )]
+    ExpectedLengthSet {
```

Comment:
Maybe this should be called `ExpectedLengthOfSet` or `ExpectedLengthSet`? I don't think it's quite grammatical right now.

Reply:
Sure thing. I think I was trying (too hard) for parity with `ExpectedSingletonSet`. I've changed this to `UnexpectedSetLength`.